### ISSUE TITLE
spell check does correctly identify abbreviations.

### ISSUE THREAD

**Issue by dnav** [typora/typora-issues#3440]
Enter the following text:
```
With abbreviation e.g. an example, there is an issue i.e. a false positive.
```

The spell checker tags `e.g.` and `i.e.` as spelling errors.

The proposed corrections are `e.g.` and `i.e.`. If you apply the correction, the sentence becomes:

```
With abbreviation e.g.. an example, there is an issue i.e.. a false positive.
```

but the abbreviations are still tagged as errors.

Typora version 0.9.86
Auto Detect Language enabled
Detected language: EN-US
Windows 10 pro version 1909
Windows display language: English (United States)

**Comment by MartinNiederl** [comment]
Any news about the problem?

**Comment by abnerlee** [comment]
relates #214

**Comment by shengjiex98** [comment]
I still have the same problem. Any updates on the fix?

**Comment by abnerlee** [comment]
fixed in 1.3.6
